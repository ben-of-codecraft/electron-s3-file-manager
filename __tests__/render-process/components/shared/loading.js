const React = require('react');
const ReactTestRenderer = require('react-test-renderer');
const Loading = require('../../../../src/renderer-process/components/shared/loading');

describe('loading', () => {
	test('render', () => {
		const render = ReactTestRenderer.create(<Loading/>);

		expect(render.toJSON()).toMatchSnapshot();
	});
});
