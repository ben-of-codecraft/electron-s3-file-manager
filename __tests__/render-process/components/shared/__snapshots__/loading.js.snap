// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loading render 1`] = `
<div
  className="text-center text-muted py-5"
>
  <div
    className="spinner-border"
  >
    <span
      className="visually-hidden"
    >
      Loading...
    </span>
  </div>
</div>
`;
